src/brain/issueLabelHandler/route.ts

```diff
@@ -113,15 +111,6 @@ export async function markUnrouted({
     body: `Assigning to @${SENTRY_ORG}/support for [routing](https://open.sentry.io/triage/#2-route), due by **<time datetime=${timeToRouteBy}>${readableDueByDate}</time> (${lastOfficeInBusinessHours})**. ⏲️`,
   });
 
-  const itemId: string = await addIssueToGlobalIssuesProject(payload.issue.node_id, repo, issueNumber, octokit);
```

Comment:
This is duplicate logic, since when `Waiting for: Support` is added, there is automation that kicks in in `followups.ts` that handles this. I have a feeling that's where this error is coming from:
https://sentry.sentry.io/issues/4239815892/?project=5246761&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=24h&stream_index=3&utc=true